lib/inspec/objects/tag.rb

```diff
@@ -0,0 +1,27 @@
+# encoding:utf-8
+
+module Inspec
+  class Tag
+    attr_accessor :key, :value
+
+    def initialize(key, value)
+      @key = key
+      @value = value
+    end
+
+    def to_hash
+      {
+        name: @name,
+        options: @opts,
+      }
+    end
+
+    def to_ruby
+      "tag #{key.inspect}: #{value.inspect}"
+    end
+
+    def to_s
+      "Tag #{@key} with #{@value}"
```

Comment:
Just a Ruby styling thing, but I try to avoid using instance variables if they're set up as attr_readers or attr_accessors, just for consistency.